pdsc/ingest.py

```diff
@@ -241,6 +264,18 @@ def ingest_idx(label_file, table_file, configpath, outputdir):
         will be stored
     """
     instrument, table = parse_table(label_file, table_file)
+
+    # erd: radius will differ if mars vs moon
+    if 'lroc' in instrument:
+        # use moon radius
+        body_radius = MOON_RADIUS_M
+        print('using Moon radius in ingest_idx: ')
+        print(body_radius)
+    else:
+        body_radius = MARS_RADIUS_M 
+        print('using Mars radius in ingest_idx: ')
+        print(body_radius)
```

Comment:
We might want to add a `body` entry in the config file; this will make it easier for people to add other lunar instruments in the future.